tests/proofs/memory-abstract-type-lemmas.k

```diff
@@ -0,0 +1,8 @@
+require "kwasm-lemmas.k"
+
+module MEMORY-ABSTRACT-TYPE-LEMMAS
+  imports KWASM-LEMMAS
+
+  rule #setRange(BM, EA, #getRange(BM, EA, WIDTH), WIDTH) => BM
+
+endmodule
```

Comment:
why is this moved to its own file?

Reply:
I want to make sure the concrete-type memory proofs go through without this, because this makes them trivial. By having this applied only to the one proof the other rules in `kwasm-lemmas` get properly exercised.

Reply:
Ah, you mean why not just a module in the same file? Good point, that's better.